examples/complete/README.md

```diff
@@ -0,0 +1,51 @@
+# atlas-basic - complete
+
+This example shows how the atlas-basic module can be used to create:
+
+- A MongoDB Atlas project.
+- A MongoDB Atlas cluster provisioned with AWS provider for the project.
+- Several IP addresses and CIDR blocks for the project.
+- Several database users for the project.
```

Comment:
I would list these items in the following order:
Project, users, IP addresses, then cluster.